### ISSUE TITLE
Error loading restraints module using API function add_module_restraints()

### ISSUE THREAD

**Issue by olibclarke** [pemsley/coot#80]
In v0.9.8.91EL-ccp4, when I try to use the function `add_module_restraints()`, either in the scripting interface or via a startup script, the following error is printed to the log and only part of the menu is loaded:

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/oc2188/.coot-preferences/coot_trimmings.py", line 86, in <module>
    add_module_restraints()
  File "/Applications/CCP4/ccp4-8.0/coot_py2/lib/python2.7/site-packages/coot/gui_prosmart.py", line 226, in add_module_restraints
    add_coot_menu_separator(menu)
NameError: global name 'add_coot_menu_separator' is not defined
```

The module loads correctly iff I launch it from `Calculate...Modules...Restraints`, and the other modules (at least refine, cryoEM and glyco) load correctly via their respective API functions.

Cheers
Oli

**Comment by pemsley** [comment]
Thanks.
Fixed in d0ff45f118b2d1f519c0ba748a53e9725f883ed3.
It will need a release of course.

**Comment by olibclarke** [comment]
Thanks Paul!
